tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrAllowed.hlsl

```diff
@@ -0,0 +1,15 @@
+// RUN: %dxc -E main -T lib_6_7 %s | FileCheck %s
+
+// This test used to fail because it didn't take into account
+// the possibility of the compute shader being a library target.
+// Now, this test is an example of what should be allowed
+// and no errors are expected.
+
+// CHECK: define void @main()
```

Comment:
I think we're banning tests that only check for main I'm afraid. Could you check that the attrib is applied?